PhysicsTools/NanoAOD/python/protons_cff.py

```diff
@@ -0,0 +1,63 @@
+import FWCore.ParameterSet.Config as cms
+from PhysicsTools.NanoAOD.common_cff import *
+from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
+from Configuration.Eras.Modifier_run2_nanoAOD_94XMiniAODv1_cff import run2_nanoAOD_94XMiniAODv1
+from Configuration.Eras.Modifier_run2_nanoAOD_94XMiniAODv2_cff import run2_nanoAOD_94XMiniAODv2
+from Configuration.Eras.Modifier_run2_nanoAOD_94X2016_cff import run2_nanoAOD_94X2016
+from Configuration.Eras.Modifier_run2_nanoAOD_102Xv1_cff import run2_nanoAOD_102Xv1
+
+filteredProtons = cms.EDProducer("PPSProtonFilter")
```

Comment:
If you use the following, you will be able to modify more easily a single (or a few) parameters in the producer whenever you'll need it:
```suggestion
import RecoPPS.ProtonReconstruction.ppsProtonFilter_cfi
filteredProtons = ppsProtonFilter.clone()
```
(or whatever you decide to rename ppsProtonFilter``